src/SourceBuild/Arcade/tools/SourceBuildArcadeTarball.targets

```diff
@@ -146,6 +146,11 @@
       Command="git submodule update --init --recursive"
       WorkingDirectory="$(TarballRepoSourceDir)" />
 
+    <!-- Remove the git objects folder to free up tarball space -->
```

Comment:
In addition to this, I think we should add `--depth 1` to the fetch command.  The size reduction on top of your proposed changes is insignificant however the time savings is significant as it reduces the network load.  On my machine it cuts the tarball creation in half.

Reply:
Thanks for the suggestion.  Fixed.